mtags/src/main/scala/scala/meta/internal/pc/MetalsGlobal.scala

```diff
@@ -384,7 +385,11 @@ class MetalsGlobal(
       case _ => code
     }
     val unit = newCompilationUnit(codeWithCursor, filename)
-    val richUnit = new RichCompilationUnit(unit.source)
+    val source =
+      if (filename.endsWith(".sc"))
```

Comment:
`filename.isScalaScript` for consistency?

Reply:
Done.